jenkins/spark-tests.sh

```diff
@@ -70,10 +70,11 @@ MORTGAGE_SPARK_SUBMIT_ARGS=" --conf spark.plugins=com.nvidia.spark.SQLPlugin \
     --class com.nvidia.spark.rapids.tests.mortgage.Main \
     $RAPIDS_TEST_JAR"
 
-# need to disable pooling for udf test to prevent cudaErrorMemoryAllocation
-CUDF_UDF_TEST_ARGS="--conf spark.rapids.python.memory.gpu.pooling.enabled=false \
-    --conf spark.rapids.memory.gpu.pooling.enabled=false \
-    --conf spark.executorEnv.PYTHONPATH=rapids-4-spark_2.12-0.3.0-SNAPSHOT.jar \
+RAPIDS_FILE_NAME=${RAPIDS_PLUGIN_JAR##*/}
```

Comment:
extra ##* ?

Reply:
That is bash black magic for `${basename $RAPIDS_PLUGIN_JAR}`

Reply:
ah ok, didn't think about that, just looked like typo, thanks